app/models/company.rb

```diff
@@ -0,0 +1,8 @@
+class Company < ApplicationRecord
```

Comment:
This is always a 1:1 relationship, so we can definitely do that!

I guess with a small app like this, we don't have to worry much about performance.

My thought was that, since we only use Stocks in the portfolio, we don't really need all the extra Company data in the portfolios and could be a separate table.

Reply:
we can make this change in a follow up PR!